cypress/integration/customizedResources/user/add/AddRecord.spec.js

```diff
@@ -0,0 +1,30 @@
+import faker from 'faker';
```

Comment:
try to stick to one convention when naming `spec` files. kebab-case or camelCase to unify all